exonum/src/runtime/rust/interfaces.rs

```diff
@@ -0,0 +1,161 @@
+// Copyright 2019 The Exonum Team
```

Comment:
Creating one file for common interfaces is OK for now, but later it'll be better to have a separate folder with `mod.rs` just exporing each interface one by one.